src/index.html

```diff
@@ -6,9 +6,108 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Kosugi&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          >
+      </a>
+
+      <div class="menu__mobile hide">
+        <img
+          src="./images/menu_icon.png"
+          alt="menu icon"
+          class="menu__mobile--icon"
+        >
+      </div>
```

Comment:
This section is out of the scope of the task

Reply:
Probably it is related to smaller screen comment - so you can leave it

Reply:
this section only for @media